Dart/resources/META-INF/plugin.xml

```diff
@@ -209,6 +209,10 @@
             text="Dart File" description="Create new Dart file">
       <add-to-group group-id="NewGroup" anchor="before" relative-to-action="NewFromTemplate"/>
     </action>
+    <action id="Dart.Reanalyze" class="com.jetbrains.lang.dart.ide.actions.ReanalyzeAction"
+            text="Reanalyze Sources" description="Reanalyze all sources">
+      <add-to-group group-id="BuildMenu" relative-to-action="CompileProject" anchor="after"/>
```

Comment:
Do you think it is important to have the action in main menu by default? Isn't it enough to have it in tool window?
Personally I'd prefer not to add it to main menu by default. Users can do it themselves in Preferences | Menu and Toolbars